src/stagedsync/stage.rs

```diff
@@ -48,3 +50,11 @@ pub struct UnwindInput {
     pub stage_progress: u64,
     pub unwind_to: u64,
 }
+
+#[derive(Error, Debug)]
+pub enum SyncError {
```

Comment:
Out of scope for staged sync framework, move into stage

Reply:
Moved to `sender_recovery.rs`